### ISSUE TITLE
Using import.meta throws an error

### ISSUE THREAD

**Issue by Eazymov** [TomerAberbach/parse-imports#6]
Code:
```javascript
import parseImports from 'parse-imports';

const code = `
  console.log(import.meta);
`;

for (const $import of await parseImports(code)) {
  console.log($import);
}
```

Error:
```
node:internal/process/esm_loader:97
    internalBinding('errors').triggerUncaughtException(
                              ^

AssertionError [ERR_ASSERTION]: false == true
    at parseModuleSpecifier (file:///.../node_modules/parse-imports/src/parse-module-specifier/index.js:27:3)
    at Object.[Symbol.iterator] (file:///.../node_modules/parse-imports/src/index.js:47:33)
    at Generator.next (<anonymous>)
    at file:///.../test.mjs:7:12 {
  generatedMessage: true,
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '=='
}
```

**Comment by TomerAberbach** [comment]
Thanks for the report! I'll take a look soon

**Comment by TomerAberbach** [comment]
Released: [v1.1.1](https://github.com/TomerAberbach/parse-imports/releases/tag/v1.1.1)
